README.md

```diff
@@ -1,17 +1,13 @@
 # Setup
-If you do not have the docker postgres image, it can be pulled from the docker container repo.
+Run a docker image with postgres.
```

Comment:
I'm a bit scared, Lee.
![Screenshot 2019-04-09 at 13 26 53](https://user-images.githubusercontent.com/6660202/55796838-2e1ca180-5acb-11e9-8be8-a865c4f828af.png)
![download](https://user-images.githubusercontent.com/6660202/55796841-2f4dce80-5acb-11e9-9b3b-43a29e735905.jpg)

Reply:
This is very concerning as I have no recollection of this.